modules/programs/pay-respects.nix

```diff
@@ -0,0 +1,65 @@
+{ config, lib, pkgs, ... }:
+with lib;
```

Comment:
```suggestion
```
Avoid nesting `with` expressions.

Reply:
Thanks for taking the time to do that code review. I've fixed most of these and haven't pushed them yet.

Could you tell me if you're referring to doing something like this?

```
let
  inherit (lib) mkEnableOption mkPackageOption getExe optionalString mkIf;

  cfg = config.programs.pay-respects;
  payRespectsCmd = getExe cfg.package;
in {
```

Similar to https://github.com/nix-community/home-manager/blob/master/modules/programs/poetry.nix

Reply:
I believe using inherit was correct and I've updated it

Reply:
Yeah, `inherit` is ideal IMO. https://nix.dev/guides/best-practices.html#with-scopes